docs/content-crag/guides/dagster/graph_job_op.mdx

```diff
@@ -372,6 +372,50 @@ This makes it easier to catch errors and makes it possible to reference the grap
 
 The same pattern works for sensors.
 
+### A pipeline with a config and a schedule
+
+With the pipeline APIs, if you want to schedule runs with a pre-defined config, you need to do something like this:
+
+```python file=/guides/dagster/graph_job_op/pipeline_with_schedule_return_config.py
+from dagster import pipeline, schedule, solid
+
+
+@solid(config_schema={"param": str})
+def do_something(_):
+    ...
+
+
+@pipeline
+def do_it_all():
+    do_something()
+
+
+@schedule(cron_schedule="0 0 * * *", pipeline_name="do_it_all")
+def do_it_all_schedule():
+    return {"solids": {"do_something": {"config": {"param": "some_val"}}}}
```

Comment:
If the config is hardcoded, it's probably preferable to supply the config when invoking `to_job`.  I think the main use case for setting the config inside the schedule body is when the config depends on the execution time (in the context).